force-app/main/default/lwc/attendanceRow/attendanceRow.js

```diff
@@ -58,7 +58,7 @@ export default class AttendanceRow extends LightningElement {
 
     @api
     getRow() {
-        return this._isEdited ? this.localRecord : null;
+        return this._isEdited || !this.recordId ? this.localRecord : null;
```

Comment:
Didn't catch this in the first iteration. If it hasn't been changed at all, which is quite possible now, given that we default both default fields, it needs to be saved even though it hasn't been flagged as edited.